### PR TITLE
feat: Fetch bindings.json from official Azure Functions repo

### LINKED ISSUE #15: Fetch bindings.json from azure fucntions templates repo
Get [bindings.json](https://github.com/serverless/serverless-azure-functions/blob/master/shared/bindings.json) from [azure templates bindings](https://github.com/Azure/azure-webjobs-sdk-templates/blob/dev/Bindings/bindings.json)

### PR DESCRIPTION
**Pull request by tbarlow12**
## What did you implement:

Fetch `bindings.json` from [Azure Functions template repo](https://github.com/Azure/azure-functions-templates)

Closes #15 

## How did you implement it:

Simple GET request to fetch the JSON object from the master branch of the GitHub repo. Still need to write tests to verify. Will mock out actual axios request in tests.

## Todos:

_**Note: Run `npm run test:ci` to run all validation checks on proposed changes**_

- [x] Ensure there are no lint errors.  
       **Validate via `npm run lint`**  
       _Note: Some reported issues can be automatically fixed by running `npm run lint:fix`_
- [x] Write tests and confirm existing functionality is not broken.  
       **Validate via `npm test`**
- [x] Write documentation
- [x] Provide verification config / commands / resources
- [x] Enable "Allow edits from maintainers" for this PR
- [x] Update the messages below

**_Is this ready for review?:_** YES 
**_Is it a breaking change?:_** NO

### CODE REVIEW

File: src/shared/bindings.ts [hunk 1]
```diff
@@ -111,4 +114,20 @@ export class BindingUtils {
...
+   */
+  private static async getBindingsJson() {
+    try {
+      const { data } = await axios({
+        method: "get",
+        url: configConstants.bindingsJsonUrl,
+      });
+      return data;
+    } catch (err) {
+      return defaultBindingsJson;
```
Comment (wbreza) [inline src/shared/bindings.ts]: Great that you kept in the default - i was thinking about the case where it goes away at some point or if there is some transient error.

File: src/config.ts [hunk 2]
```diff
@@ -41,6 +41,7 @@ export const configConstants = {
     prefix: "sls",
     localPort: 7071,
   },
+  bindingsJsonUrl: "https://raw.githubusercontent.com/Azure/azure-functions-templates/master/Functions.Templates/Bindings/bindings.json"
```
Comment (wbreza) [inline src/config.ts]: Can you update our default file with the latest from the remote version?

Comment (tbarlow12) [reply src/config.ts]: Sorry, I should've specified in the description, that's included. It's a huge file, so Github doesn't show the diff by default

Comment (wbreza) [reply src/config.ts]: Ah, i thought it would have shown...

**Comment by pjlittle** [comment]
good stuff.

**Pull request merged**